### PR TITLE
Add EIP: NFT Fusing Protocol

### PR DESCRIPTION
**Pull request by saitama2009**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by saitama2009** [comment]
> The commit [238ca06](https://github.com/ethereum/EIPs/commit/238ca061e6d4d8aaa78c111f0386f0adf81e9851) (as a parent of [267f3e6](https://github.com/ethereum/EIPs/commit/267f3e61239b7c9d0122401a8b87b6b3f79c3bb5)) contains errors. Please inspect the [Run Summary](https://github.com/ethereum/EIPs/pull/6927/files) for details.

 @axic @Pandapip1 @SamWilsn @xinbenlv  I saw the branch was behind the master then updated it. But why `eipbot/6927` always renames the filename back to `eip-draft_nft_fusing_protocol.md`, and how to fix it?

File: EIPS/eip-6927.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+## Motivation
+
+The current NFT ecosystem lacks a standardized approach to combine multiple NFTs into a new one. While some projects have implemented custom solutions, these are often ad hoc and project-specific, limiting their interoperability with other NFTs and platforms. This standard aims to create a standardized method for fusing NFTs, enabling users to create new NFTs by combining existing ones according to customizable combination rules. This will foster more incredible innovation and creativity within the NFT space and create new possibilities for NFT use cases.
+
+## Specification
+
+The keywords “MUST,” “MUST NOT,” “REQUIRED,” “SHALL,” “SHALL NOT,” “SHOULD,” “SHOULD NOT,” “RECOMMENDED,” “MAY,” and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+**Interface**
```
Comment (SamWilsn) [inline EIPS/eip-6927.md:26]:
```suggestion
### Interface
```

File: EIPS/eip-6927.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+* Minting a valid combo with ERC-721 and/or ERC-1155 NFTs.
+* Dismantling a combo and returning locked NFTs to their owners.
+* Setting combo rules and ensuring compliance during the minting - - process.
+* Approving ERC-721 NFTs for use in combos.
+* Querying combo rules and ingredients.
+
+## Reference Implementation 
+
+An implementation of this EIP will be provided upon acceptance of the proposal.
+
```
Comment (SamWilsn) [inline EIPS/eip-6927.md:155]:
```suggestion
<!--

## Test Cases

Test cases should be developed to cover the following scenarios:

* Minting a valid combo with ERC-721 and/or ERC-1155 NFTs.
* Dismantling a combo and returning locked NFTs to their owners.
* Setting combo rules and ensuring compliance during the minting - - process.
* Approving ERC-721 NFTs for use in combos.
* Querying combo rules and ingredients.

## Reference Implementation 

An implementation of this EIP will be provided upon acceptance of the proposal.

-->
```

These sections are optional, and if you use an HTML-style comment, the linter will make sure you come back to them before advancing statuses.

File: EIPS/eip-6927.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+eip: 6927
+title: NFT Fusing Protocol
+description: Fusing multiple NFTs into a new one.
+author: Saitama (@saitama2009), Combo <combo@1combo.io>, Luigi <luigi@1combo.io>, Gary <gary.y.mao@gmail.com>, Ansen <anssen712@yahoo.com>
+discussions-to: https://ethereum-magicians.org/t/draft-eip-nft-fusing-standard/13959
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-23
+requires: 165, 721
```
Comment (SamWilsn) [inline EIPS/eip-6927.md:11]:
```suggestion
requires: 165, 721, 1155
```

`Ingredient.amount` doesn't make sense unless you explicitly mention ERC-1155.

File: EIPS/eip-6927.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    /// to participate in the combo and the rules that apply.
+    /// A customized combination rule consists of several ComboFactors.
+    /// @param collectionAddress The NFT collection address
+    /// @param minRequired Minimum number of NFTs required from the collection
+    /// @param maxAllowed Maximum number of NFTs allowed from the collection
+    /// @param lockTokens Indicates if tokens from the collection will be locked when minting a combo
+    /// @param maxReuseCount Maximum total reuse count of a token from the collection in
+    /// this combo collection (only applicable for ERC-721 and when lockTokens is false)
+    struct ComboFactor {
+        address collectionAddress;
```
Comment (SamWilsn) [inline EIPS/eip-6927.md:73]: You should mention somewhere what standards the `collectionAddress` is allowed to be. I'm guessing it's ERC-721 and ERC-1155 from your Rationale section?

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**